api/src/main/kotlin/com/playkids/onboarding/api/sqs/SQSEventEmitter.kt

```diff
@@ -0,0 +1,26 @@
+package com.playkids.onboarding.api.sqs
+
+import com.playkids.onboarding.sqs.SQSEmitter
+import com.playkids.onboarding.sqs.extensions.toMessageAttributeValue
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.future.await
+import kotlinx.coroutines.launch
+import software.amazon.awssdk.services.sqs.SqsAsyncClient
+import software.amazon.awssdk.services.sqs.model.SendMessageRequest
+
+class SQSEventEmitter(
+    private val sqs: SqsAsyncClient,
+    private val queueUrl: String
+): SQSEmitter {
+
+    override fun sendEvent(message: String, attributes: Map<String, String>) = GlobalScope.launch {
```

Comment:
Sometimes it is better to pass the scope in which things are going to run as a parameter, leaving things more customizable and preventing overuse of GlobalScope. It works, though, so you can leave it like this.